build-yaml/econet-etwh-esp32.yaml

```diff
@@ -6,8 +6,16 @@ substitutions:
   board: esp32dev
 
 packages:
-  econet-etwh: !include ../econet_electric_tank_water_heater.yaml
+  econet:
+    url: https://github.com/esphome-econet/esphome-econet
+    ref: main
```

Comment:
Use github_ref substitution? Same elsewhere.

Reply:
Unfortunately you can't use substitutions for `packages:` as per https://github.com/esphome/issues/issues/4939